tests/unit/distributed/test_model_utils.py

```
@@ -956,3 +961,384 @@ def test_distributed_logprob_all_tests(
+                        cutoff_idx = len(sorted_probs)
+
+                    # Indices that should be kept
+                    kept_indices = sorted_indices[:cutoff_idx]
+
+                    # Verify that kept indices are not -inf in the final result
+                    for idx in kept_indices:
+                        # Let's just check: if it was in our calculated "kept set", it should be kept.
+                        if not torch.isinf(filtered_logits[b, s, idx]):
+                            continue
+
+                        assert False, f"Index {idx} in top-p should not be -inf"
```

Ruff (0.14.6)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)